ui/components/Lifecycle/Environments/index.js

```diff
@@ -296,6 +296,7 @@ const Environments = ({ organization, classes }) => {
       
       Learn more about the behavior of [lifecycle of environments and their resources](https://docs.meshery.io/concepts/logical/environments) in Meshery Docs.`,
       variant: PROMPT_VARIANTS.DANGER,
+      helpText: `Deletes the "${environment.name}" Environment`,
```

Comment:
not required, the environment name here, its been already shown inside modal

Reply:
Sure, will remove the variable, and helper prompt will display **Deletes the Environment**, would that be okay?